sdk/src/main/java/com/qonversion/android/sdk/dto/QonversionError.kt

```diff
@@ -2,7 +2,8 @@ package com.qonversion.android.sdk.dto
 
 data class QonversionError(
     val code: QonversionErrorCode,
-    val additionalMessage: String = ""
+    val additionalMessage: String = "",
+    internal val httpCode: Int? = null
```

Comment:
We agreed, that you'll also remove purchase callback with HTTP code and use this field there